### ISSUE TITLE
Optimize pack200 verification and installation

### ISSUE THREAD

**Issue by tschulte** [threerings/getdown#96]
When using pack200, all packed jar files are unpacked twice:

* download packed jar file to `.jar.pack.gz_new`
* unpack `.jar.pack.gz_new` to `.jar.pack.gz_new.tmp`
* verify `.jar.pack.gz_new.tmp`
* delete `.jar.pack.gz_new.tmp`
* rename `.jar.pack.gz_new.tmp` file to `.jar.pack.gz`
* unpack `.jar.pack.gz` to `.jar`

Verify (Steps 2-4) happens parallelized, which is good. Install (Step 5-6) doesn't happen parallelized. But since the install steps unpack the packed files (again), the upgrade takes longer than it needs.

In my case with 250 packed jar files, the verification takes 10s, and install takes 22s. The unpacked file from the verification phase should not be deleted and be re-used in the installation phase.

**Comment by tschulte** [comment]
I am working on this right now

**Comment by vgropp** [comment]
Hi, what is the status of this issue? As i experience slow performance using pack200 with latest head

**Comment by tschulte** [comment]
Sorry, I had to stop working on it, but forgot to say that here.

**Comment by sdgx** [comment]
Note that pack200 has been deprecated in Java 11 (released today), and is scheduled for removal from the Java runtime in a subsequent release.

http://openjdk.java.net/jeps/336

**Comment by tschulte** [comment]
Yes, I know. In our project, the difference between using pack200 and not using it is 30MB/79MB. But pack200 has it's drawbacks as well. Especially when it comes to signing, etc.

I don't know if pack200 will be ditched completely, or if maybe https://dev.karakun.com/webstart/ will reimplement it. But for this issue I think it is best to wait to see if pack200 will somehow survive, and if not, remove that functionality completely.

**Comment by Halahala1993** [comment]
Are there any plans for getdowns use of pack200 now that it's been removed from Java 14? 

https://openjdk.java.net/jeps/367
